migration/index.html

```diff
@@ -321,6 +342,12 @@ <h2 id="test">6. Test</h2>
     <li>The #laminas-issues channel of the <a href="https://laminas.dev/chat/">Laminas Slack</a>.</li>
 </ul>
 
+<h2 id="post-migration">Post migration</h2>
+
+<p>In case the <code>--keep-package-versions</code> flag was used, you definitely want to diff the <code>composer.json</code> and restore the old package constraints and remove the packages which were not required on project level before migration. After cleaning up the <code>composer.json</code>, you have to update the <code>composer.lock</code> by using the following command:
+        <pre class="codehilite"><code class="language-bash">$ composer update --lock</code></pre>
+</p>
```

Comment:
If this step is optional then we should mark it like step four "Verify changes".

Reply:
It is optional for now. I will add `optional` to the Text but not to the id to ensure deep-linking wont break if there are probably more steps to do after migration in the future.

Or should this library considered as `final`?

Reply:
I've pushed the changes, thus, I've added the `optional` information to both the hashbang and the text.
If there will be a non-optional section called "post migration" in the future, one can simply add another section.